src/bitcoin-tx.cpp

```diff
@@ -667,11 +667,13 @@ static void MutateTx(CMutableTransaction& tx, const std::string& command,
         MutateTxDelOutput(tx, commandVal);
     else if (command == "outaddr")
         MutateTxAddOutAddr(tx, commandVal);
-    else if (command == "outpubkey")
+    else if (command == "outpubkey") {
+        if (!ecc) { ecc.reset(new Secp256k1Init()); }
```

Comment:
Sorry,.. missed the fact that `std::unique_ptr<Secp256k1Init> ecc;` initialises `ecc` with `nullptr`